components/modules/Kconfig

```diff
@@ -164,6 +164,13 @@ menu "NodeMCU modules"
             Includes the QR Code Generator from
             https://www.nayuki.io/page/qr-code-generator-library
 
+    config NODEMCU_CMODULE_RMT
+        bool "Remote Control pulse generator/receiver"
+        default "n"
+        help
+            Includes the RMT module to use the ESP32's built-in 
```

Comment:
Unless the Lua name of the module is (uppercase) RMT, it should probably say "rmt" here.

Reply:
done